src/Core/Commander/Providers/TileProvider.js

```diff
@@ -40,57 +34,23 @@ TileProvider.prototype.preprocessLayer = function preprocessLayer(/* layer*/) {
     /* no-op */
 };
 
-TileProvider.prototype.getGeometry = function getGeometry(bbox, cooWMTS) {
-    var geometry;
-    var n = Math.pow(2, cooWMTS.zoom + 1);
-    var part = Math.PI * 2.0 / n;
-
-    if (this.cacheGeometry[cooWMTS.zoom] !== undefined && this.cacheGeometry[cooWMTS.zoom][cooWMTS.row] !== undefined) {
-        geometry = this.cacheGeometry[cooWMTS.zoom][cooWMTS.row];
-    } else {
-        if (this.cacheGeometry[cooWMTS.zoom] === undefined) {
-            this.cacheGeometry[cooWMTS.zoom] = [];
-        }
-
-        var precision = 16;
-        var rootBBox = new BoundingBox(0, part + part * 0.01, bbox.south(), bbox.north());
-
-        geometry = new TileGeometry(rootBBox, precision, this.ellipsoid, cooWMTS.zoom);
-        this.cacheGeometry[cooWMTS.zoom][cooWMTS.row] = geometry;
-    }
-
-    return geometry;
-};
-
-
 TileProvider.prototype.executeCommand = function executeCommand(command) {
-    var bbox = command.paramsFunction.bbox;
+    var bbox = command.bbox;
 
-    // TODO not generic
-    var tileCoord = this.projection.WGS84toWMTS(bbox);
     var parent = command.requester;
 
     // build tile
-    var geometry; // getGeometry(bbox,tileCoord);
```

Comment:
I'm not against remove `getGeometry()`.
**Note for later** : it's possible to get identical tile's geometry #229